fhir-bucket/src/main/java/com/ibm/fhir/bucket/client/FhirClient.java

```diff
@@ -291,9 +290,11 @@ public FhirServerResponse post(String url, String body) {
                 logger.fine(msg.toString());
             }
 
-            // If we are posting a bundle, then we need to parse the response entity
-            boolean isBundle = url.isEmpty();
-            return buildResponse(response, startTime, isBundle);
+            // If we are posting a bundle or calling a custom operation, 
```

Comment:
Per convention please upper case this as `FHIRClient`
I'd also suggest giving this a slightly different name `FHIRBucketClient`

Reply:
I guess this comment belongs at the class level...i was confused at first so thought I'd make that clear.

Reply:
done